### DISCUSSION TITLE
[Bug] 客户端异常

### DISCUSSION THREAD

**Discussion by shaoqing404** [lobehub/lobehub#1176] [General Question] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Docker

### 🌐 浏览器

Edge

### 🐛 问题描述

发生反馈异常
![LobeChat_Stable Diffusion 提示词专家_2024-01-29](https://github.com/lobehub/lobe-chat/assets/121151546/693506e6-92f2-4557-83a8-560109c0c6dd)

### 🚦 期望结果

在错误的位置时应当截断

### 📷 复现步骤

无法确定触发原因。

### 📝 补充信息

无相关的内容补充

[a comment by an automated account is not shown]

**Comment by mushan0x0** [comment, marked as answer]
这些设置最好不要更改，不然会发生重复的情况
<img width="1222" alt="image" src="https://github.com/lobehub/lobe-chat/assets/29084441/8eaa353c-e443-4d33-820c-63e258737a9b">
